### PR TITLE
Added to Sections - Content Callout:

### PR DESCRIPTION
**Pull request by prd409-utsa**
3 variants: white (default), blue, orange

Screen Shot 2022-08-03 at 5 46 10 PM

Screen Shot 2022-08-03 at 5 46 27 PM

Screen Shot 2022-08-03 at 5 46 41 PM

### CODE REVIEW

File: components/01-visual-styling/02-button/01-text-link/text-link.scss [hunk 1]
```diff
@@ -11,7 +11,7 @@
     color: $white;
 }
 
-.blue-bg h2 {
+a.blue-bg h2 {
```
Comment (lalov1) [inline components/01-visual-styling/02-button/01-text-link/text-link.scss:14]: It looks like the text isn't showing up on the blue banner:
https://utsa-asc.github.io/college-dls/431-dls-full-width-content-callout-no-image-no-cta/components/detail/text-link.html

**Review by lalov1 — COMMENTED** [review]
It seems like in the mock-up that the default banner is a grey that matches the site footer. Also, please see notes about the missing header after the CSS changes for the text link section.

File: components/01-visual-styling/02-button/01-text-link/text-link.scss (hunk 1 above)
Comment (prd409-utsa) [reply components/01-visual-styling/02-button/01-text-link/text-link.scss:14]: The issue stated make default white.

**Review by Uri-E — COMMENTED** [review]
I think it looks gewd, but as Lallo said the first variant should have the grey background. Also, the headers should be H2 (heading) and H3 (subheading) since the page title should have the only H1.

File: components/01-visual-styling/02-button/01-text-link/text-link.scss (hunk 1 above)
Comment (prd409-utsa) [reply components/01-visual-styling/02-button/01-text-link/text-link.scss:14]: I'll just add a gray variant as well. Problem between mockup examples and stated issue requirements solved!

Comment (lalov1) [reply components/01-visual-styling/02-button/01-text-link/text-link.scss:14]: Sounds good! I wasn't sure if I missed something with the conversation.

**Comment by prd409-utsa** [comment]
Additional Gray variant added. h1 and h2 changed to h2 and h3 respectively.

**Review by Uri-E — APPROVED** [review]
Looks gewd!

**Review by lalov1 — APPROVED** [review]
Look great! 👍🏽

**Pull request merged**